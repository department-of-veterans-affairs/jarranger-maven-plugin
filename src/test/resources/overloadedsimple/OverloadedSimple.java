@SuppressWarnings("all")
public final class OverloadedSimple {
  public void setClientInfo(String key, String value) {}

  public void setClientInfo(Object p) {}

  public String getClientInfo(String name) {
    return null;
  }

  public Object getClientInfo() {
    return null;
  }
}
