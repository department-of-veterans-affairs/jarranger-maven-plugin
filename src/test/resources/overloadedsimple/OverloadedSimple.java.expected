@SuppressWarnings("all")
public final class OverloadedSimple {
    public String getClientInfo(String name) {
        return null;
    }

    public Object getClientInfo() {
        return null;
    }

    public void setClientInfo(Object p) {
    }

    public void setClientInfo(String key, String value) {
    }
}
